apps/wallet-mobile/src/navigation.tsx

```diff
@@ -457,6 +462,15 @@ export const useWalletNavigation = () => {
     })
   }
 
+  const navigateToGovernanceCenter = () => {
+    navigation.navigate('app-root', {
+      screen: 'governance',
+      params: {
+        screen: 'center',
+      },
+    })
+  }
```

Comment:
I think this wrapper for the routes was originally for the tabs (a kind of root) but it doesn't seem like a bad idea to have the routes in the same place!